src/Wallabag/ApiBundle/Controller/WallabagRestController.php

```diff
@@ -122,11 +122,11 @@ public function postEntriesAction(Request $request)
             $this->get('wallabag_core.content_proxy')->assignTagsToEntry($entry, $tags);
         }
 
-        if (true === (bool) $isStarred) {
+        if ($isStarred === 'true') {
```

Comment:
In that case, we should also change values retrieved to 0/1, what do you think ?